src/remote-docker/remote-docker.yml

```diff
@@ -1,4 +1,4 @@
-# Orb Version 0.1.9
+# Orb Version 0.1.10
```

Comment:
Wonder if there is a bug in the publish-orb script...
```
artsy/remote-docker@0.1.9 is the latest, cannot publish older version 0.1.1
Please update ./src/remote-docker/remote-docker.yml to have a version greater than 0.1.9
```

Reply:
Hmmm

Reply:
Oh, yep! Yikes.